include/xkbcommon/xkbcommon-names.h

```diff
@@ -34,9 +34,13 @@
 #define XKB_MOD_NAME_SHIFT      "Shift"
 #define XKB_MOD_NAME_CAPS       "Lock"
 #define XKB_MOD_NAME_CTRL       "Control"
-#define XKB_MOD_NAME_ALT        "Mod1"
-#define XKB_MOD_NAME_NUM        "Mod2"
-#define XKB_MOD_NAME_LOGO       "Mod4"
+#define XKB_MOD_NAME_ALT        "Alt"
+#define XKB_MOD_NAME_META       "Meta"
+#define XKB_MOD_NAME_LOGO       "Super"
+#define XKB_MOD_NAME_GUI        "Super"
+#define XKB_MOD_NAME_SUPER      "Super"
+#define XKB_MOD_NAME_HYPER      "Hyper"
+#define XKB_MOD_NAME_NUM        "NumLock"
```

Comment:
hmm, I wonder if that and the `ALT`/`LOGO` changes are an API/ABI break...

Reply:
They may be breaking changes in the case that the modifier mapping is different.

For `Alt`, I found only 2 vendor layouts in xkeyboard-config which does not map to `Mod1`:
- `symbols/fujitsu_vndr/us`
- `symbols/hp_vndr/us`

For `NumLock`, I found only 2 vendor layouts in xkeyboard-config which does not map to `Mod2`:
- `symbols/fujitsu_vndr/us`
- `symbols/digital_vndr/pc`

I did not find other mapping for `Super` than `Mod4`.

The other possible issue is when these modifiers are not even mapped! But with the base layout `pc` they should be mapped by default.